modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/Utils.scala

```diff
@@ -100,9 +100,16 @@ object Utils {
         password.value.value.toCharArray
       )
 
-  def randomString(length: Int): String = {
+  def randomString(input: String, length: Int): String = {
+    val hash = MessageDigest.getInstance("SHA-256")
```

Comment:
Maybe you can reuse tessellation fn?

Reply:
Can you tell which function? I mean the file with this function, I haven't seen this function yet

Reply:
Maybe Hash.hashCodeFromBytes? but not sure if  it fits your purpose.

Reply:
I think it would be Hash.fromBytes(index.getBytes("utf-8")) then use .value to read out the string of the hash.